### PR TITLE
Add Zenbeats.app latest

### PR DESCRIPTION
**Pull request by claui**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/zenbeats.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  name 'Zenbeats'
+  homepage 'https://www.roland.com/us/products/zenbeats/'
+
+  pkg 'Zenbeats_Installer.pkg'
+
+  uninstall pkgutil: 'jp.co.roland.zenbeats.Zenbeats.pkg',
+            quit:    'jp.co.roland.zenbeats',
+            signal:  [
+                       ['KILL', 'jp.co.roland.zenbeats'],
+                     ]
```
Comment (vladimyr) [inline Casks/zenbeats.rb:15]:
```suggestion
            signal:  ['KILL', 'jp.co.roland.zenbeats'],
```

File: Casks/zenbeats.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  zap trash: [
+               '~/Library/Caches/com.juce.locks/juceAppLock_Roland Zenbeats',
+               '~/Library/Caches/jp.co.roland.zenbeats',
+               '~/Library/Saved Application State/jp.co.roland.zenbeats.savedState',
```
Comment (vladimyr) [inline Casks/zenbeats.rb:20]: You should also delete the shared library

```suggestion
               '/Users/Shared/Zenbeats Library',
               '~/Library/Caches/com.juce.locks/juceAppLock_Roland Zenbeats',
               '~/Library/Caches/jp.co.roland.zenbeats',
               '~/Library/Saved Application State/jp.co.roland.zenbeats.savedState',
```

File: Casks/zenbeats.rb (hunk 1 above)
Comment (claui) [reply Casks/zenbeats.rb]: Changed, thanks!

File: Casks/zenbeats.rb (hunk 2 above)
Comment (claui) [reply Casks/zenbeats.rb:18]: I’d rather not because it’s where user data goes.

Comment (vladimyr) [reply Casks/zenbeats.rb:18]: That is true but `zap` is never performed automatically and it is specifically intended to remove all traces of an app except files created by the user. ([docs](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md))
As far as I understand the library contains downloaded samples from Roland's repository and not user-created files?
image

Comment (claui) [reply Casks/zenbeats.rb:18]: I don’t know. The library has subdirectories like `Songs` and `Tracks` so I figured I’d err on the side of caution.

Comment (claui) [reply Casks/zenbeats.rb:18]: There’s [some limited evidence](https://amchannel.info/C0kz2bS_2xvtoEdXH-OUoKw) that people seem to be aware of the `Library` directory. Looks like they keep some content there that may not be available for re-download if `zap`ped.

Comment (vladimyr) [reply Casks/zenbeats.rb:18]: Well, in that case, ignore it 👍

**Review by vladimyr — APPROVED** [review]

**Pull request merged**